docker/prometheus/prometheus.yml

```diff
@@ -0,0 +1,32 @@
+# my global config
```

Comment:
Is there any configuration to commit in the config git repository; https://github.com/spring-petclinic/spring-petclinic-microservices-config ?

Reply:
Nice catch! There is. I'll fork that and create a PR for those changes.